models/dpr_test/model.json

```diff
@@ -0,0 +1,40 @@
+{
+    "system": {
+        "target": "audiomini",
+        "sampleClockFrequency": 48000,
+        "systemClockFrequency": 98304000,
+        "processing": "sample",
+        "audioIn": {
+            "wordLength": 32,
+            "fractionLength": 28,
+            "signed": true,
+            "numberOfChannels": 2
+        },
+        "audioOut": {
+            "wordLength": 32,
+            "fractionLength": 28,
+            "signed": true,
+            "numberOfChannels": 2
+        }
+    },
+
+    "devices": [
+        {
+            "name": "dpr_test",
+            "registers": [
+                {
+                    "dataType": {
+                        "fractionLength": 0,
+                        "signed": false,
+                        "type": "sfdfs",
```

Comment:
Is this a real data type? I know we don't use the type field...

Reply:
So we do ever so slightly use/check this field in Matlab parsing and if you don't have the field, then it errors. I am updating the parsing so it isnt an issue but I would guess that is why there is a random string there for now

Reply:
I updated the format to more closely represent the datatype being passed